services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.js

```diff
@@ -105,7 +104,7 @@ const CatalogMultiselectFilter = ({
       </button>
       <PopoverContent className={styles.content} ref={contentRef}>
         {/* div wrapper addeed to receive focus */}
-        {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
+        {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex -- TODO: add role */}
```

Comment:
Worked with Andrea on this and we concluded that we need guidance from the system squad or accessibility team on what role this container should have (as well as its children)

Reply:
Opened #666 to track